### PR TITLE
Issue #SB-14332 feat: export content refactor to sync file structure with content import and mobile app

### PR DESCRIPTION
**Pull request by anoophm**

### CODE REVIEW

File: server/controllers/content/content.ts [hunk 1]
```diff
@@ -255,169 +256,50 @@ export default class Content {
...
-                    error.message
-                    } `
-                );
+            );
+            childNode = dbChildResponse.docs;
+        }
+        const contentExport = new ExportContent(content, childNode);
+        contentExport.export((err, data) => {
+            if (err) {
                 res.status(500);
```
Comment (HarishGangula) [inline server/controllers/content/content.ts:242]: Log the  error

File: server/controllers/content/content.ts [hunk 2]
```diff
@@ -255,169 +256,50 @@ export default class Content {
...
                 return res.send(Response.error('api.content.export', 500));
             }
-        })();
+            console.log('Ecar exported successfully', data);
```
Comment (HarishGangula) [inline server/controllers/content/content.ts]: remove console logs

File: server/manager/contentExportManager/contentExport.ts [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    this.parentArchive = archiver('zip', { zlib: { level: 9 }});
+  }
+  join(...paths){
+    return path.join(...paths);
+  }
+  public async export(cb){
+    await fileSDK.mkdir('temp');
+    this.cb = cb;
+    try {
+      this.parentManifest = await fse.readJson(this.join(this.contentBaseFolder, this.dbParentDetails.identifier,  'manifest.json'));
```
Comment (HarishGangula) [inline server/manager/contentExportManager/contentExport.ts]: use fileSDK

File: server/manager/contentExportManager/contentExport.ts [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+  contentBaseFolder = fileSDK.getAbsPath('content');
+  parentArchive;
+  parentManifest;
+  ecarName;
+  corruptContents = [];
+  startTime = Date.now();
+  cb;
+  parentDetails;
+  constructor(public dbParentDetails, dbChildNodes){
+    this.parentArchive = archiver('zip', { zlib: { level: 9 }});
```
Comment (HarishGangula) [inline server/manager/contentExportManager/contentExport.ts]: move to fileSDK

File: server/manager/contentExportManager/contentExport.ts [hunk 5]
```diff
@@ -0,0 +1,200 @@
+import * as fs from 'fs';
+import * as archiver from 'archiver';
+import * as path from 'path';
+import * as  _ from 'lodash';
+import * as fse from 'fs-extra';
+import { manifest } from '../../manifest';
+import { containerAPI } from 'OpenRAP/dist/api';
+let fileSDK = containerAPI.getFileSDKInstance(manifest.id);
+import { logger } from '@project-sunbird/ext-framework-server/logger';
+
```
Comment (HarishGangula) [inline server/manager/contentExportManager/contentExport.ts:9]: Write comments to each method

**Review by HarishGangula — CHANGES_REQUESTED** [review]

File: server/controllers/content/content.ts (hunk 1 above)
Comment (anoophm) [reply server/controllers/content/content.ts:242]: contentExport.export has error log

**Review by HarishGangula — APPROVED** [review]

**Pull request merged**